frontend/src/bundles/common/pages/base.tsx

```diff
@@ -476,6 +477,42 @@ const Base: React.FC = () => {
                     ))}
                 </div>
             </div>
+            <div style={{ backgroundColor: '#EFF3FF', padding: '0 1rem 1rem' }}>
```

Comment:
Use className instead of 'style'

Reply:
Is this a test page? The budget cards are demonstrated here so as not to renders the entire component of the budget page.